backend/coins/btc/handlers/handlers.go

```diff
@@ -364,6 +364,52 @@ func (handlers *Handlers) getAccountBalance(*http.Request) (interface{}, error)
 	}, nil
 }
 
+type slip24Request struct {
+	RecipientName string `json:"recipientName"`
+	Nonce         string `json:"nonce"`
+	Memos         []struct {
+		Type         string `json:"type"`
+		Text         string `json:"text"`
+		Refund       string `json:"refund,omitempty"`
+		CoinPurchase struct {
+			CoinType int    `json:"coinType"`
+			Amount   string `json:"amount,omitempty"`
+			Address  string `json:"address,omitempty"`
+		} `json:"coinPurchase"`
+	} `json:"memos,omitempty"`
+	Outputs []struct {
+		Amount  float64 `json:"amount"`
+		Address string  `json:"address"`
+	} `json:"outputs,omitempty"`
+	Signature string `json:"signature,omitempty"`
```

Comment:
why omitEmpty in all the fields? This skips the fields if empty when marshaling, but we don't serialize this to JSON, right?

Reply:
Right, I'll remove it.